samcli/lib/bootstrap/bootstrap.py

```diff
@@ -51,54 +51,47 @@ def get_current_account_id(profile: Optional[str] = None):
 
 def _get_stack_template():
     gc = GlobalConfig()
-    info = {"version": __version__, "installationId": gc.installation_id if gc.installation_id else "unknown"}
-
-    template = """
-    AWSTemplateFormatVersion : '2010-09-09'
-    Transform: AWS::Serverless-2016-10-31
-    Description: Managed Stack for AWS SAM CLI
-
-    Metadata:
-        SamCliInfo: {info}
-
-    Resources:
-      SamCliSourceBucket:
-        Type: AWS::S3::Bucket
-        Properties:
-          VersioningConfiguration:
-            Status: Enabled
-          Tags:
-            - Key: ManagedStackSource
-              Value: AwsSamCli
-
-      SamCliSourceBucketBucketPolicy:
-        Type: AWS::S3::BucketPolicy
-        Properties:
-          Bucket: !Ref SamCliSourceBucket
-          PolicyDocument:
-            Statement:
-              -
-                Action:
-                  - "s3:GetObject"
-                Effect: "Allow"
-                Resource:
-                  Fn::Join:
-                    - ""
-                    -
-                      - "arn:"
-                      - !Ref AWS::Partition
-                      - ":s3:::"
-                      - !Ref SamCliSourceBucket
-                      - "/*"
-                Principal:
-                  Service: serverlessrepo.amazonaws.com
-                Condition:
-                  StringEquals:
-                    aws:SourceAccount: !Ref AWS::AccountId
-
-    Outputs:
-      SourceBucket:
-        Value: !Ref SamCliSourceBucket
-    """
-
-    return template.format(info=json.dumps(info))
+    template = {
+        "AWSTemplateFormatVersion": "2010-09-09",
+        "Transform": "AWS::Serverless-2016-10-31",
+        "Description": "Managed Stack for AWS SAM CLI",
+        "Metadata": {
```

Comment:
Should we add the metadata ourselves in code, and move this entire template to its own file. We can then run cfn-lint as part of CI process on this template.